ChangeLog.rst

```diff
@@ -15,6 +15,8 @@ Features added
 :ghpull:`193` - update versions of `kube-prometheus`, Elasticsearch and
 Kubespray
 
+:ghissue:`143` - Capture Nginx (ingress) metrics (:ghpull:`189`, :ghpull:`203`)
```

Comment:
Nit-picky, and may be done in a later PR, but for consistency, shouldn't this be

```
:ghpull:`189`, :ghpull:`203` - collect `nginx-ingress` metrics and deploy a dashboard (:ghissue:`143`)
```